data/json/items/gun/44paper.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "id": "colt_army",
+    "type": "GUN",
```

Comment:
>How do I remove mod slots from a copied weapon, for example with the pepperbox?

Look at the Steyr AUG in 223.json: You override them. I think mugling meant that fields like "symbol" "skill" and "color" can be removed, since they are specified by "revolver".